lib/fulfil/client.rb

```diff
@@ -93,6 +93,8 @@ def put(model: nil, id: nil, endpoint: nil, body: {})
       uri = URI(model_url(model: model, id: id, endpoint: endpoint))
 
       result = request(verb: :put, endpoint: uri, json: body)
+      return result if result.first.is_a?(Integer)
```

Comment:
I don't think this is the right point for fixing this. It should be addressed in the parser instead of having this odd check here. Mind taking a stab at it?

Reply:
Definitely